README.md

```diff
@@ -42,6 +42,7 @@ To add your own project below, just [edit](https://github.com/wwdc/2020/edit/mas
 |[Fernando Fontecha](http://ferfontecha.com)||[YouTube](https://www.youtube.com/watch?v=zi2J60IKbKw)|UIKit, Core Animation, PlaygroundSupport|Submitted|
 |[Frank Foster](https://twitter.com/frankefoster)|[Github](https://github.com/analogpotato/WWDCSubmission)||AVFoundation, Vision, VisionKit|Submitted|
 |[Glenn Brannelly](http://glennbrannelly.com/)||[YouTube](https://youtu.be/lQAr6TQetm4)|SpriteKit, SceneKit|Submitted|
+|[Haotian Zheng](http://fincher.im/)|[GitHub](https://github.com/JustinFincher/WWDC19Playground)||UIKit Dynamics, SwiftUI, SpriteKit, AVFoundation|Accepted|
```

Comment:
Please only include up to four technologies in your entry (also you link to a WWDC19 playground, is that intentional?)

Reply:
Sorry that I linked to a wrong repo. I have now pushed another commit and changed it back, thank you!